tests/test_edit_functions.py

```diff
@@ -33,7 +73,74 @@ def test_translate_local_multiple_results(
     assert translate_local(test_search_hangul) == expected_english_res
 
 
-# TODO: translate_local failure case.
+def test_english(mocked_translate_local):
+    mwo_밥_first = MockWordObject()
+    mwo_밥_second = MockWordObject()
+
+    mwo_밥_first.english = "a meal"
+    mwo_밥_second.english = "rice"
+
+    mock_translate_local_results = [mwo_밥_first, mwo_밥_second]
+
+    mocked_translate_local.return_value = mock_translate_local_results
+
+    expected_built_str = "a meal\n<br>rice"
+
+    assert english("밥") == expected_built_str
+
+
+def test_hanja(mocked_translate_local):
+    # these hanja are unrelated, but used for unittesting purposes
+
+    mwo_밥_first = MockWordObject()
+    mwo_일월_second = MockWordObject()
+
+    mwo_밥_first.english = "食"
+    mwo_일월_second.english = "一月"
+
+    mock_translate_local_results = [mwo_밥_first, mwo_일월_second]
+
+    mocked_translate_local.return_value = mock_translate_local_results
+
+    expected_built_str = "食,一月"
```

Comment:
Is this test missing an assert or am I not understanding?

Reply:
You are totally right, great catch haha

Reply:
I also missed the hanja attributes too. 🤦

Reply:
Think I got it this time though!